protoc-gen-kotlin/lib/src/commonMain/kotlin/pbandk/gen/Main.kt

```diff
@@ -32,6 +32,9 @@ fun runGenerator(request: CodeGeneratorRequest): CodeGeneratorResponse {
 
         val needToGenerate = request.fileToGenerate.contains(protoFile.name)
 
+        // TODO figure out how to connect this to a config or something.
+        val generateJson = true
```

Comment:
The `protoc-gen-kotlin` options are passed in via `request.parameter` up on line 16 of this file. See the way `jsonUseProtoNames` is defined at the top of the `CodeGenerator` class for an example.

To pass the parameter, you'd call `protoc-gen-kotlin` like this:

```sh
protoc --kotlin_out=generate_json=false:build/generated/proto some.proto
```

Or if you're using the protobuf gradle plugin, you'd use the `option` command to pass this parameter (example below is using Gradle Kotlin DSL, but there's a similar syntax for the Gradle Groovy DSL):

```kt
...
    task.plugins {
        id("kotlin") {
            option("generate_json=false")
        }
    }
```